src/Compilers/Core/Portable/PEWriter/PdbMetadataWrapper.cs

```diff
@@ -114,7 +114,7 @@ internal unsafe interface IMetaDataImport
         uint FindMemberRef(uint td, string stringName, [MarshalAs(UnmanagedType.LPArray, SizeParamIndex = 3)] byte[] voidPointerSigBlob, uint byteCountSigBlob);
         uint GetMethodProps(uint mb, out uint pointerClass, IntPtr stringMethod, uint cchMethod, out uint pchMethod, IntPtr pdwAttr,
           IntPtr ppvSigBlob, IntPtr pcbSigBlob, IntPtr pulCodeRVA);
-        unsafe uint GetMemberRefProps(uint mr, ref uint ptk, StringBuilder stringMember, uint cchMember, out uint pchMember, out byte* ppvSigBlob);
+        uint GetMemberRefProps(uint mr, ref uint ptk, StringBuilder stringMember, uint cchMember, out uint pchMember, out byte* ppvSigBlob);
         uint EnumProperties(ref uint handlePointerEnum, uint td, uint* arrayProperties, uint countMax);
```

Comment:
The interface is already marked `unsafe` and there are many other unsafe methods here.